### PR TITLE
Update bug labels.md

### PR DESCRIPTION
**Pull request by kierachell**
connects to discussion about:
- https://github.com/department-of-veterans-affairs/appeals-design-research/pull/154
- https://github.com/department-of-veterans-affairs/appeals-design-research/blob/master/Best%20Practices%20for%20Creating%20Issues/MarkDown%20Template%20-%20Bug.md

will help up prioritize bugs to work on

PS: if this PR fails because of Travis... 😞

### CODE REVIEW

File: docs/labels.md [hunk 1]
```diff
@@ -2,7 +2,10 @@ From now, if you want to add a label, you need to run it by the team, so everyon
 
 ##Labels Definitions
 - **discussion** An issue used for the team to discuss solutions and ideas related to a high level problem. The team can discuss on the issue for some amount of time, and possibly come together in person to determine the outcome.
-- **bug** - A bug that has been found with code in production. Any bugs with stories that have not yet been completed should be handled on that issue.
+- **bug-high-priority** - A bug that has been found with code in production. Data loss, data corruption or system unavailable  
+- **bug-medium-priority** - A bug that has been found with code in production. Important functionality is unavailable but there is a workaround
+- **bug-low-priority** - A bug that has been found with code in production. Secondary functionality is unavailable and there is a workaround
+- **bug-ui** - A bug that has been found with code or design in production. Cosemtic issues that do not affect functionality.
```
Comment (shanear) [inline docs/labels.md]: can we merge `bug-low-priority` and `bug-ui`, a lot of times a bug in the ui is due to something not in the ui. It's kind of a misleading and hard to use category. Plus, there are some UI bugs that are higher priority, we shouldn't pigeonhole them

Comment (kierachell) [inline docs/labels.md]: https://github.com/department-of-veterans-affairs/appeals-design-research/pull/154#discussion_r109429613

🤕

Comment (shanear) [inline docs/labels.md]: 😬 we should take bug-ui out of that too.

**Comment by kierachell** [comment]
@shanear I combined `low-priority` with `ui` is that the intent?...confused now

**Pull request merged**